### PR TITLE
Configurable tasks and specs paths

### PR DESCRIPTION
**Pull request by rafalsobota**

### CODE REVIEW

**Comment by rafalsobota** [comment]
I signed it!

File: src/queue.js [hunk 1]
```diff
@@ -122,12 +120,25 @@ function Queue() {
     logger.debug('Queue(): Error during initialization', error);
     throw new Error(error);
   }
+  
+  if (_.isPlainObject(constructorArguments[0])) {
+    if (_.isUndefined(constructorArguments[0].tasksRef) || _.isUndefined(constructorArguments[0].specsRef)) {
```
Comment (drtriumph) [inline src/queue.js]: check out lodash's `_.has()` method here to simplify the logic

File: src/queue.js [hunk 2]
```diff
@@ -122,12 +120,25 @@ function Queue() {
...
+      error = 'When queueRef is an object then it must specify tasksRef and specsRef.';
```
Comment (drtriumph) [inline src/queue.js]: Grammar: `When queueRef is an object it must contain both a tasksRef and a specsRef`

**Comment by drtriumph** [comment]
Thanks for the PR. I like the idea of being able to specify these values and how this is backwards compatible, but the JSDoc comments, README documentation, and tests all need updating before I can consider merging this PR.

Also check out my couple of nit-pick comments.

**Comment by rafalsobota** [comment]
Thanks for response. Something else should be improved?

**Comment by steida** [comment]
Will be this PR ever merged?

**Pull request closed without merging**